de-lib/src/main/java/org/iplantc/de/diskResource/client/views/widgets/DiskResourceDotMenu.java

```diff
@@ -65,6 +66,9 @@ void addMenuItems(DiskResource diskResource) {
 
     void checkButtonStatus(DiskResource diskResource) {
         shareBtn.setEnabled(isOwner(diskResource));
+        if (diskResource instanceof File) {
+            dataLinkBtn.setEnabled(isOwner(diskResource));
```

Comment:
The service level only checks for the user having write permissions; presumably this should match (unless the previous code only allowed this for ownership, in which case I guess we should retain that?)

Reply:
Yeah, the previous code (and I tested it in prod just to be sure) only enables that option with `own`, which is strange if the service is different.  At least for now, the old and new match which I think is ok.